openassessment/xblock/schema.py

```diff
@@ -169,5 +169,6 @@ def datetime_validator(value):
                 })
             ]
         })
-    ]
+    ],
+    Required('show_rubric_during_response'): bool,
```

Comment:
Should this have a default value? Or will older XBlocks automatically populate this value on change?
```suggestion
    Required('show_rubric_during_response', default=False): bool,
```

Reply:
I believe it will be automatically populated, but there's no harm in being more explicit